scrapers/ManyVids/ManyVids.py

```diff
@@ -0,0 +1,332 @@
+    """
+    if details:
+        details = re.sub(r"\\", "", details)
+        details = bs(details, features='lxml').get_text()
+    return details
+
+
+def map_ethnicity(ethnicity: str) -> str:
+    if ethnicity is None:
+        return None
+
+    ethnicities = {
+            "Alaskan": "alaskan",
+            "Asian": "asian",
+            "Black / Ebony": "black",
+            "East Indian": "east indian",
+            "Latino / Hispanic": "hispanic",
+            "Middle Eastern": "middle eastern",
+            "Mixed": "mixed",
+            "Native American": "native american",
+            "Pacific Islander": "pacific islander",
+            "White / Caucasian": "white",
+            "Other": "other"
+    }
+
+    found = ethnicities.get(ethnicity)
+    if found:
+        return found
+    return ethnicity
+
+def get_scene(scene_id: str) -> dict:
+    """
+    Get and parse scene meta from manyvid's video player json api
+    """
+    try:
+        response = get_request(
+            f"https://video-player-bff.estore.kiwi.manyvids.com/videos/{scene_id}"
+        )
+    except requests.exceptions.RequestException as api_error:
+        log.error(f"Error {api_error} while requesting data from manyvids api")
+        return None
+
+    meta = response.json()
+    scrape = {}
+    scrape['title'] = meta.get('title')
+    scrape['details'] = meta.get('description')
+    if meta.get('modelId'):
+        model_name = get_model_name(meta['modelId'])
+        if model_name:
+            scrape['performers'] = []
+            scrape['performers'].append({'name': model_name})
+    scrape['image'] = meta.get('thumbnail')
+    date = meta.get('launchDate')
+    if date:
+        date = re.sub(r"T.*", "", date)
+        scrape['date'] = date
+    if meta.get('tags'):
+        scrape['tags'] = [{"name": x} for x in meta['tags']]
+
+    return scrape
+
+
+def get_model_bio(url_handle: str, performer_url: str) -> dict:
+    """
+    Get and parse model profile from manyvid's json endpoint
+    """
+    try:
+        response = get_request(
+            f"https://bkxljkxlbh.execute-api.us-east-1.amazonaws.com/prod/profiles/{url_handle}"
+        )
+    except requests.exceptions.RequestException as api_error:
+        log.error(f"Error {api_error} while requesting data from manyvids api")
+        return None
+    model_meta = response.json()
+    log.debug(json.dumps(model_meta)) # useful to get all json entries
+    scrape = {}
+    scrape['name'] = model_meta.get('displayName')
+    scrape['image'] = model_meta.get('portrait')
+    log.debug(f"image {scrape['image']}")
+    date = model_meta.get('dob')
+    if date:
+        date = re.sub(r"T.*", "", date)
+        scrape['birthdate'] = date
+
+    description = model_meta.get('description')
+    scrape['details'] = clean_text(description)
+
+    scrape['gender'] = model_meta.get('identification')
+    if scrape['gender'] and scrape['gender'] == "Trans":
```

Comment:
don't need the `if scrape['gender']`, only  `scrape['gender'] == "Trans"` is needed